start

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 npm start
```

Comment:
maybe comment this line saying that npm start will run the node.js wifi connection app and only run the lines after this when a wifi connection is made. Some people dont know what npm is :/